Makefile

```diff
@@ -216,7 +216,7 @@ debug:
 	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build -mod vendor -gcflags "-N -l" -ldflags="-X '$(BUILDINFO_PKG).Version=$(RELEASE_VERSION)' -X '$(BUILDINFO_PKG).Revision=$(RELEASE_REVISION)'" -a -o bin/$(CMD) $(MAIN_GO_FILE)
 
 .PHONY: dev
-dev: prereqs docker-generate compile-for-coverage
+dev: prereqs generate compile-for-coverage
```

Comment:
for development, I think we should just use the local development clang/llvm for speed purposes, rather than having to use the docker generate.